quickstart/.travis.yml

```diff
@@ -1,5 +1,13 @@
 sudo:       required
```

Comment:
Same here as in [the comment in the other file](https://github.com/Strauman/travis-latexbuild/pull/17/files#r223842870)